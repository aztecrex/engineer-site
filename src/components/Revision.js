import React from 'react';

class Revision extends React.Component {
  render() {
    return <span className='revision'>3</span>
  }
}

export default Revision;
