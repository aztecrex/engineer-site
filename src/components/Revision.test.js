import React from 'react';
import ezJson from 'enzyme-to-json';

import Revision from './Revision';

describe('render', () => {
  it('renders', () => {
    // when
    const rendered = shallow(<Revision />);

    //then
    expect(ezJson(rendered)).toMatchSnapshot();
  });
});
