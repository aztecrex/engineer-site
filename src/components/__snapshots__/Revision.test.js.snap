exports[`render renders 1`] = `
<span
  className="revision">
  9
</span>
`;
